__tests__/unit/flex.test.js

```diff
@@ -19,16 +21,19 @@ describe('Test processBeforeValidate and processAfterValidate invocations', asyn
             must_exclude:
               regex: wip|work in progress
   `
-  let configWithMultiple = config + `
+  let configWithMultipleString = configString + `
       - when: pull_request_review.submitted
         validate:
           - do: milestone
             no_empty:
               enabled: true
   `
 
+  let config = yaml.safeLoad(configString)
```

Comment:
I'm using `safeLoad` across the tests just to maintain the YML here for readability. Let me know if you'd prefer something else, e.g. just putting the objects here directly.

Reply:
This is a good proposal and allows for us to easily manipulate the config in a test and re-mock. However, I'm not seeing this need at the moment in the current tests where `Helper. mockConfigWithContext` is used.

Any reason for wanting this as an object assigned to the variable `config` within the tests?

At the moment I'm deducing that you want it to be obvious that it is `YML`.

Reply:
> Any reason for wanting this as an object assigned to the variable config within the tests?

Hmm, great point. I updated `mockConfigWithContext` to mock `probotContext.config` which returns an object, so that's why. But now that you mentioned it I noticed this would be much simpler if I just flipped it around and made `mockConfigWithContext` continue taking a string, and just do the `yaml.safeLoad` once inside the `probotContext.config` mock.

That way we can continue using strings in the tests and avoid a lot of these changes.

Have updated this now. Let me know what you think :slightly_smiling_face: